### PR TITLE
Added info about how counters work

### PR DESCRIPTION
**Pull request by rrg92**
Issue #4236 asked more details about buffer cache hit ratio. Then, I add a lot of more information to users be able to guide users where it can find more details about how performance counter works, from Windows WMI docs!

### CODE REVIEW

**Comment by PRMerger7** [comment]
@rrg92 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/relational-databases/performance-monitor/use-sql-server-objects.md [hunk 1]
```diff
@@ -38,12 +38,12 @@ ms.author: jrasnick
...
   
+ SQL Server performance counters values uses the Windows Performance Counter engine. Some values are not calculated directly by SQL Server engine. SQL Server just provide base values to performance counter engine and it will do the work for calculation (like show percentages, etc.). The [sys.dm_os_performance_counters &#40;Transact-SQL&#41;](../../relational-databases/system-dynamic-management-views/sys-dm-os-performance-counters-transact-sql.md) dynamic management view provide all counters with original value generated by SQL Server, and the column `cntr_type` indicates the type of counter. How performance counter engine will process this value depend on this type. For more detailed about these types, [check WMI documentation about performance counters](https://docs.microsoft.com/en-us/windows/win32/wmisdk/wmi-performance-counter-types)
```
Comment (pmasl) [inline docs/relational-databases/performance-monitor/use-sql-server-objects.md:41]: Please replace SQL Server with the token `[!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)]`, and SQL Server engine with `[!INCLUDE[ssDEnoversion](../../includes/ssdenoversion-md.md)]`. 

Suggested redaction:

```
[!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] performance counters values are generated using the Windows Performance Counter (WPC) engine. Some counter values are not calculated directly by [!INCLUDE[ssDEnoversion](../../includes/ssdenoversion-md.md)]. The [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] provides base values to the WPC engine which will perform the required calculations (like percentages). The [sys.dm_os_performance_counters &#40;Transact-SQL&#41;](../../relational-databases/system-dynamic-management-views/sys-dm-os-performance-counters-transact-sql.md) dynamic management view provides all counters with the original value generated by [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)], and the column `cntr_type` indicates the type of counter. How the WPC engine processes [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] counter values depends on this type. For more details about performance counter types, see the [WMI documentation](https://docs.microsoft.com/en-us/windows/win32/wmisdk/wmi-performance-counter-types).
```

**Comment by PRMerger7** [comment]
@rrg92 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/relational-databases/performance-monitor/use-sql-server-objects.md (hunk 1 above)
Comment (rrg92) [reply docs/relational-databases/performance-monitor/use-sql-server-objects.md]: Your suggestions was applied. I didn't know that we must use this, and this is nice. I will pay more attention on next submissions. Thanks.

File: docs/relational-databases/performance-monitor/use-sql-server-objects.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ ms.author: jrasnick
...
+[!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] performance counters values are generated using the Windows Performance Counter (WPC) engine. Some counter values are not calculated directly by [!INCLUDE[ssDEnoversion](../../includes/ssdenoversion-md.md)]. The [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] provides base values to the WPC engine which will perform the required calculations (like percentages). The [sys.dm_os_performance_counters &#40;Transact-SQL&#41;](../../relational-databases/system-dynamic-management-views/sys-dm-os-performance-counters-transact-sql.md) dynamic management view provides all counters with the original value generated by [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)], and the column `cntr_type` indicates the type of counter. How the WPC engine processes [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] counter values depends on this type. For more details about performance counter types, see the [WMI documentation](https://docs.microsoft.com/en-us/windows/win32/wmisdk/wmi-performance-counter-types).
```
Comment (pmasl) [inline docs/relational-databases/performance-monitor/use-sql-server-objects.md:41]: One last thing, please remove the `en-us/` from the link, and we're done. Thanks!

**Comment by PRMerger8** [comment]
@rrg92 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/relational-databases/performance-monitor/use-sql-server-objects.md (hunk 2 above)
Comment (rrg92) [reply docs/relational-databases/performance-monitor/use-sql-server-objects.md]: Nice. Removed.

**Comment by ktoliver** [comment]
@pmasl - Is the PR approved for merging? Thanks.

**Comment by pmasl** [comment]
@ktoliver yes it is now

**Comment by ktoliver** [comment]
> yes it is now

@pmasl Awesome, thanks.

**Pull request merged**